docs/reference/objects/page.md

```diff
@@ -24,6 +24,16 @@ The site's domain.
 
 `=> example.com`
 
+## `page.params`
+{: .d-inline-block }
+object
+{: .label .fs-1 }
+
+The query params included in the current URL.
+
+For example, given the URL https://my-url.com?page=4,
+using {% raw %}`{{page.params.page}}`{% endraw %} would return "4".
```

Comment:
Any value in explaining nested params like `search[departure_month]=1235`

Reply:
Nice suggestion. But I think the docs are probably ok for this. If a developer did want to use nested params, it would just work as they expect. Probably don't need to mention it here.